Makefile

```diff
@@ -101,6 +101,7 @@ pause-container:
 	@docker rmi -f "amazon/amazon-ecs-build-pause-bin:make"
 
 pause-container-release: pause-container
+	mkdir -p "$(shell pwd)/out"
```

Comment:
`$(shell pwd)/` is redundant. You're referencing it relative to the working directory in the next line, so you might as well do it here.